### PR TITLE
Update HDS to version 3.9.0 and update Navigation/Footer

### PR DESCRIPTION
**Pull request by vincit-matu**
## 🛠️ Changelog
[//]: # "Describe the changes in this pull request here."

- Bumps all HDS dependencies to the most recent version (3.9.0), for `admin-ui`, `ui`, and `common`
- Remakes the headers (= `Navigation.tsx`) and footers for both admin and ui pages
- Removes `MainMenu` and the Breadcrumbs component from the admin-ui. The new Navigation makes them redundant.

## 🧪 Test plan
[//]: # "Help your fellow reviewer and write a short description of what's the fastest way to test your changes."

- Check out the logged in/out versions of the headers and footers on both sides, in all screen sizes
- A lot was taken care of, but there are for sure a lot of smaller stylistic issues which need fixing (e.g. table width limits in admin-ui, now that they have the full page width to utilize). Non-breaking bugs are to be fixed in a separate ticket.
- The current version should offer the same functionality as the we previously had, even if some stuff is a bit differently organized

## 🎫 Tickets
[//]: # "This pull request resolves all or part of the following ticket(s)"

- TILA-2991

### CODE REVIEW

File: apps/admin-ui/src/component/Navigation.tsx [hunk 1]
```diff
@@ -1,131 +1,229 @@
...
-// NOTE requires both client and react-router context
-const NavigationWithRouter = ({ apiBaseUrl }: Pick<Props, "apiBaseUrl">) => {
-  const history = useNavigate();
   return (
-    <Navigation apiBaseUrl={apiBaseUrl} onLogoClick={() => history("/")} />
+    <Header>
+      <ActionBar
+        title={t("common:applicationName")}
+        titleAriaLabel={t("common:applicationName")}
+        frontPageLabel={t("common:gotoFrontpage")}
```
Comment (joonatank) [inline apps/admin-ui/src/component/Navigation.tsx:177]: this is missing in https://github.com/City-of-Helsinki/tilavarauspalvelu-ui/blob/0325933f0d65eabd92834907ef93162e22cc814e/apps/admin-ui/src/i18n/index.ts#L21

need to add the same translations to `admin-ui` as `ui`

File: apps/admin-ui/src/component/Navigation.tsx [hunk 2]
```diff
@@ -1,131 +1,229 @@
...
+            style={{ filter: "invert(1)" }}
+          />
+        }
+        logoAriaLabel="Varaamo logo"
+        logoHref="/"
```
Comment (joonatank) [inline apps/admin-ui/src/component/Navigation.tsx:190]: Does this work? not require `/kasittely`? i.e. `env.NEXT_PUBLIC_BASE_URL`

**Review by joonatank — COMMENTED** [review]
Nice. A few questions / changes.

File: apps/admin-ui/src/component/Navigation.tsx [hunk 3]
```diff
@@ -1,131 +1,226 @@
...
...zItMS4wMTguNjgyIDAgLjU3MyAxLjAxOC42MzUgMi4xNjIuOTkxIDEuMjA4LjM3MiAyLjMyLjkxNSAyLjMyIDIuMjk0IDAgMS41MTgtMS40NDYgMi40MTctMy4xMTUgMi40MTctMS44MTEgMC0zLjI0Mi0uNzQ0LTMuODc3LTEuOTUybDEuODktMS4wMzljLjI0LjgyMi45MjIgMS40NDEgMS45NTUgMS40NDEuNjIgMCAxLjA1LS4yNDggMS4wNS0uNzQzem0tNi44ODItOC42NzdoLTIuMTc3djguNjkyYzAgLjc3NS4xNzUgMS4zNDguNTA5IDEuNzA1LjM1LjM1Ni44OS41MjYgMS42MzYuNTI2LjI1NSAwIC41MjUtLjAzLjc4LS4wNzcuMjctLjA2Mi40NzYtLjE0LjY1LS4yMzNsLjE5MS0xLjQyNWEyLjA3IDIuMDcgMCAwMS0uNDYuMTI0Yy0uMTI4LjAzLS4yODcuMDMtLjQ2MS4wMy0uMjg2IDAtLjQxNC0uMDc3LS41MDktLjIxNi0uMTExLS4xNC0uMTU5LS4zODctLjE1OS0uNzQ0di04LjM4MnptLTcuMjQ2IDQuNTdjLS43OTUgMC0xLjQ0Ni41NTgtMS42MjEgMS41ODFoMy4wNWMuMDE3LS44OTktLjU4Ny0xLjU4LTEuNDMtMS41OHptMy4zNTMgMy4wMDdIMjMuNjNjLjA5NSAxLjIyNC43OTQgMS44MjggMS43IDEuODI4LjgxIDAgMS4zNjctLjUyNyAxLjQ5NC0xLjI0bDEuODI4IDEuMDA3Yy0uNTQuOTYxLTEuNyAxLjc5OC0zLjMyMiAxLjc5OC0yLjE2IDAtMy43NS0xLjQ3Mi0zLjc1LTMuOTUxIDAtMi40NjQgMS42Mi0zLjk1MSAzLjcwMy0zLjk1MSAyLjA4MSAwIDMuNDY0IDEuNDQgMy40NjQgMy40ODYtLjAxNi42MDQtLjExMSAxLjAyMy0uMTExIDEuMDIzem0tMTEuMDc3IDMuMjA3aDIuMjU3VjEwLjkxNmgtMi4yNTd2NC4xMDdoLTQuMjQzdi00LjA5MUgxMS4wNnYxMC4zNjZoMi4yNTZ2LTQuMjkyaDQuMjQzdjQuMjkyeiIKICAgICAgICBmaWxsPSJjdXJyZW50Q29sb3IiIC8+Cjwvc3ZnPg=="
```
[the remark this answers is not in the document]
Comment (vincit-matu) [reply apps/admin-ui/src/component/Navigation.tsx:185]: Yeah, that's probably a good idea 😆

File: apps/admin-ui/src/component/Navigation.tsx (hunk 2 above)
Comment (vincit-matu) [reply apps/admin-ui/src/component/Navigation.tsx:190]: fixed

File: apps/admin-ui/src/component/Navigation.tsx (hunk 1 above)
Comment (vincit-matu) [reply apps/admin-ui/src/component/Navigation.tsx:177]: Removed the Footer entirely from `admin-ui`

**Review by joonatank — APPROVED** [review]
Great 👍

**Pull request merged**